packages/admin/.env.example

```diff
@@ -9,4 +9,7 @@ BACKEND_HOST=http://localhost:8080
 COOKIE_SECRET=E6AAAB815903D8CDD81246EFC8275C13EE34541134092A635572C8F7F47448BE
 SPOTLIGHT_URL=https://cabinetoffice-spotlight.force.com/s/login/
 TINYMCE_API_KEY=
-FEATURE_ADVERT_BUILDER=enabled
\ No newline at end of file
+FEATURE_ADVERT_BUILDER=enabled
+ONE_LOGIN_ENABLED=true
```

Comment:
if we set this to `false` instead it is going to be truthy when we check it inside the `isLoggedIn`function (it's a string)